src/main/webapp/i18n/de/exam.json

```diff
@@ -54,6 +54,7 @@
             "notConfirmed": "Bitte setze einen Haken in das Kästchen um fortzufahren.",
             "falseName": "Der angegebene Name ist nicht korrekt. Bitte versuche es erneut!",
             "notSet": "Nicht gesetzt",
+            "startExamToolTip": "Button ist 5 Minuten vor der Klausur aktiviert.",
```

Comment:
```suggestion
            "startExamToolTip": "Button wird 5 Minuten vor der Klausur aktiviert.",
```

Reply:
The button is shown when it is not enabled, so i propose to use future here because it **is** not activated yet